pkgs/applications/misc/sioyek/default.nix

```diff
@@ -17,15 +18,21 @@
 
 stdenv.mkDerivation rec {
   pname = "sioyek";
-  version = "1.4.0";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "ahrm";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-F71JXgYaWAye+nlSrZvGjJ4ucvHTx3tPZHRC5QI4QiU=";
+    sha256 = "sha256-wvlidoHalMJ60IQ0OPMpZKUZNgoqcFyRVgXdjL65oAU=";
   };
 
+  # fix build with mupdf 1.20
+  patches = [(fetchurl {
+    url = "https://aur.archlinux.org/cgit/aur.git/plain/mupdf-1.20.patch?h=sioyek&id=35cc1e31c4659ce16655bbb27582503ce66f334c";
```

Comment:
Upstream has fixed this and we should prefer patchs in the upstream repo instead of aur.

Bumping to that commit can be another choice.